dvc/testing/benchmarks/cli/commands/test_repro.py

```diff
@@ -0,0 +1,25 @@
+def test_repro(bench_dvc, tmp_dir, dvc, dataset):
+    dvc.add(dataset)
+    source_path = dataset.relative_to(tmp_dir).as_posix()
+    dvc.stage.add(
+        name="copy-1",
+        deps=[source_path],
+        outs=["copy-1"],
+        cmd=f"cp -R {source_path} copy-1",
+    )
+    dvc.stage.add(
+        name="copy-2",
+        deps=[source_path],
+        outs=["copy-2"],
+        cmd=f"cp -R {source_path} copy-2",
+    )
+    combine_cmds = [
+        "mkdir combine",
+        "cp -R copy-1 combine/1",
+        "cp -R copy-2 combine/2",
```

Comment:
Ideally, if we can, this and commands above should be cross-platform. That's why we have our own python scripts in regular tests. Simple python scripts would be great here.